tldw_Server_API/tests/Audiobooks/unit/test_audiobook_media_source.py

```
@@ -0,0 +1,73 @@
+
+pytestmark = pytest.mark.unit
+
+
+def test_load_source_text_uses_managed_media_database_for_media_id(
+    monkeypatch: pytest.MonkeyPatch,
+    tmp_path: Path,
+):
+    events = []
+
+    class _FakeDb:
+        def get_media_by_id(self, media_id: int):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `get_media_by_id`

(ANN202)